blocks/hero-block/index.story.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Hero from "./features/hero/default";
+
+// for more info on storybook and using the component explorer
+// https://storybook.js.org/
+export default {
+	title: "Blocks/Hero",
+	parameters: {
+		chromatic: { viewports: [320, 1200] },
+	},
+};
+
+export const showHeading = () => <Hero customFields={{ showHeading: true }} />;
+
+export const hideHeading = () => <Hero customFields={{ showHeading: false }} />;
```

Comment:
I know this is a wip block. but it's a good idea to think about storybook as well here? Personally I start with storybook then use live fusion. but everyone's process is different

Reply:
Check it out now